### DISCUSSION TITLE
cMake error in MacOs

### DISCUSSION THREAD

**Discussion by violetrose16** [fontforge/fontforge#5164] [Q&A]
Hi I am not experienced to use programming on Apple mac mini (Ventura 13.0.1, 22A400) since 2 years ago but I am learning programming.  I typed "git clone ...git then enter (terminal) as I followed INSTALL.md file 
cd fontforge
mkdir build
cd build
cmake -GNinja ..
 but make failed as errors.  Here is Error message from terminal: 
CMake Error in gdraw/CMakeLists.txt:
  Imported target "GDK3::GDK3" includes non-existent path

    "/Library/Developer/CommandLineTools/SDKs/MacOSX11.sdk/usr/include/ffi"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.

-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
violetrose@rose-Mac-mini build % >....                                            
  * An install or uninstall procedure did not complete successfully.    
  
  * The installation package was faulty and references files it does not
  provide.

  
-- Generating done-- Configuring done
CMake Error in gdraw/CMakeLists.txt:                                       
  Imported target "GDK3::GDK3" includes non-existent path
  
    "/Library/Developer/CommandLineTools/SDKs/MacOSX11.sdk/usr/include/ffi"
  
  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:
  
  * The path was deleted, renamed, or moved to another location.
  
  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.

CMake Generate step failed.  Build files cannot be regenerated correctly.
---------------------------------------------------------
--Latest version FontForge when I cloned git.  
--Ventura 13.0.1, 22A400 on Mac Mini M1 

How I can solve them or add installing SDK (command tools)? 
Here I checked [x}
- [ ] Screenshot
- [ x] The FontForge version and the operating system you're using
- [ x] The behavior you expect to see, and the actual behavior
- [ x] Steps to reproduce the behavior 
- [x] \(optional) Possible solution/fix/workaround

### When you open an issue for a change/improvement/feature request:
- [x] A description of the problem you're trying to solve, including _why_ you think this is a problem
- [x] If the feature changes current behavior, reasons why your solution is better
- [x] \(optional) Possible solution/fix/workaround

Can you help me? Thank you.

**Comment by jtanx** [comment]
Are you using homebrew? Do you have gtk3 installed?

**Reply by violetrose16** [reply]
Yes, I am using homebrew. I finished installing gtk3 with brew but still not working. I show you a screenshot. 
![Screenshot 2022-12-30 at 7 08 13 AM](https://user-images.githubusercontent.com/24404667/210085085-2d1bb6a8-7932-46f9-91c3-4a7eed7e28e2.png)

**Reply by violetrose16** [reply]
I saw that Mac0SX11.sdk folder is not in there.
